src/open_inwoner/accounts/management/commands/delete_invitations.py

```diff
@@ -0,0 +1,35 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from open_inwoner.utils.logentry import system_action
+
+from ...models import Invite
+
+
+class Command(BaseCommand):
+    help = (
+        "Delete contact invitations which are older than a predefined period "
+        "(the default is defined in `settings.INVITE_EXPIRY_DAYS`)"
+    )
+
+    def handle(self, *args, **kwargs):
+        now = timezone.now()
+        expired_invitations = Invite.objects.select_related("inviter").filter(
+            created_on__lt=now - timedelta(days=settings.INVITE_EXPIRY_DAYS)
+        )
+        inviters = set(invite.inviter for invite in expired_invitations)
+
+        for user in inviters:
+            my_expired_invitations = expired_invitations.filter(inviter__id=user.id)
+            invites_info = [
+                f"{invite.invitee_email} (invited on {invite.created_on.strftime('%Y-%m-%d')})"
+                for invite in my_expired_invitations
+            ]
+            message = f"Expired contact invitations from {user.first_name} {user.last_name} deleted"
```

Comment:
*If* we print names we use the methods so we don't forget infixes etc (eg: `user.get_full_name()`).

In this case the name might not be needed as we already set the user on the record. For triage it'd also be helpful to include a count in the message (eg: "Expired N contact invitations')